drivers/virtualbox/virtualbox_test.go

```diff
@@ -29,3 +226,27 @@ func TestGetRandomIPinSubnet(t *testing.T) {
 		t.Fatalf("expected third octet of %d; received %d", testIP[2], newIP[2])
 	}
 }
+
+func newDriver(name string) *Driver {
+	return NewDriver(name, "")
+}
+
+// create parses the provided args and tries to configure the driver for
+// a `create` action.
+func create(args []string) (*Driver, error) {
```

Comment:
I'm still adamant about removing this in favor of just creating the `Driver` structs directly.  All references to `codegangsta/cli` in the driver code are being removed in the plugins work so we shouldn't introduce any more, and we should stick to testing strictly what's in this package.